ipatests/test_webui/test_translation.py

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Test translations
+"""
+
+from ipatests.test_webui.ui_driver import UI_driver
+from ipatests.test_webui.ui_driver import screenshot
+
+try:
+    from selenium.webdriver.common.by import By
+    from selenium.webdriver.support.wait import WebDriverWait
+    NO_SELENIUM = False
+except ImportError:
+    NO_SELENIUM = True
+
+import pytest
+from re import sub
+from lxml import html
+from ipalib import api, util
+
+
+@pytest.mark.tier1
+class TestConfigPage(UI_driver):
+    """
+    Test translation of pages which are located at /ipa/config/
+    """
+
+    def get_config_url(self):
+        """
+        Get FreeIPA Web UI config url
+        """
+        host = self.config.get('ipa_server')
```

Comment:
Why wouldn't override get_base_url and keep all magic under the hood in UI_driver?

Reply:
I use the magic of ui_driver as much as i can :)
But some ( almost all? ) of this magic is related to the IPA web app and is not suited to use it outside of. 

I didn't override 'get_base_url' because it wasn't necessary. There was no benefit.
'get_config_url' is used in method 'load_url', which is not the same as ui_driver's 'load' one.

Reply:
Please, take a look at the latest version.

Reply:
Should i make something similar for #2109 ?

Reply:
Yes, if there is some common initialization.